src/app.js

```diff
@@ -0,0 +1,7 @@
+import * as config from './config/config.json';
+import {Product, User} from './models';
+
+console.log(config.name);
+
+console.log(new User());
```

Comment:
there's no need of extra `console.log` call as you're calling it from constructor of class when creating an instance